netlify/remix-compute
# chore: set up cypress

## Description

Sets up Cypress for end to end testing (E2E)

## Related Tickets & Documents



- Related Issue #
- Closes #24

Disregard the failed deployment preview

## QA Instructions, Screenshots, Recordings

1. Pull down this branch, e.g. via the GitHub CLI, `gh co 25`
2. Eun `npm install`
3. Run `npm run e2e`
4. The Cypress UI opens mentioning that E2E is configured.

![CleanShot 2023-02-03 at 13 47 11](https://user-images.githubusercontent.com/833231/216682979-d25d4ccc-5716-42b6-be24-478b6de67c92.png)

For us to review and ship your PR efficiently, please perform the following steps:

- [x] Open a [bug/issue](https://github.com/netlify/cli/issues/new/choose) before writing your code 🧑‍💻. This ensures we
      can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or
      something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
      guide and passes our tests.

**A picture of a cute animal (not mandatory, but encouraged)**

![A hamster trying to fit into a paper towel role](https://media.giphy.com/media/hyZLXtoAYenCw/giphy.gif)
